posts/2024/sdk_12_4_0_ga.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Titanium SDK 12.4.0.GA released'
+date: '2024-07-17'
+category: 'Release'
+author: 'Hans Knöchel'
+teaser: 'Android API 34 Support'
+image: https://tidev.io/images/titanium-general.png
+---
+
+![Titanium SDK 12.4.0.GA](/images/titanium-general.png)
+
+The next version of the Titanium SDK is now is available: <b>12.4.0.GA</b>. Titanium SDK 12.4.0 is a minor release of the SDK, adding new features and platform updates.
+
+This update is focussed on supporting the latest Android API level 34, which will be required by Google for all new apps and app updates starting in August 2024.
```

Comment:
```suggestion
This update is focused on supporting the latest Android API level 34, which will be required by Google for all new apps and app updates starting in August 2024.
```
Both are correct according to Google but I think one `s` is the preferred (US) spelling